integration-tests/src/test/java/oracle/weblogic/kubernetes/ItCrossDomainTransaction.java

```diff
@@ -300,31 +361,29 @@ public void testCrossDomainTransaction() {
       logger.info("curl command returned {0}", result.toString());
       assertTrue(result.stdout().contains("Status=Committed"), "crossDomainTransaction failed");
     }
-
   }
 
   /*
-   * This test verifies cross domain transaction is successful and able to re-establish connection when
-   * one domain is shutdown. Domain in image with wdt is used to create 2 domains in different namespaces.
-   * A servlet is deployed to the admin server of domain1. This servlet starts a transaction with
-   * TMAfterTLogBeforeCommitExit transaction property set. The servlet inserts data into oracleDB table and
-   * sends a message to a JMS queue as part of a same transaction.The coordinator (server in domain2)
-   * should exit before commit and the domain1 admin server should be able to re-establish connection
-   * with domain2 and the transaction should commit.
+   * This test verifies a cross-domain transaction with re-connection. 
+   * It makes sure the disitibuted transaction is completed successfully 
+   * when a coordinator server is re-started after writing to transcation log
+   * A servlet is deployed to the admin server of domain1. 
+   * The servlet starts a transaction with TMAfterTLogBeforeCommitExit 
+   * transaction property set. The servlet inserts data into an Oracle DB 
+   * table and sends a message to a JMS queue as part of the same transaction. 
+   * The coordinator (server in domain2) should exit before commit and the 
+   * domain1 admin server should be able to re-establish the connection with 
+   * domain2 and the transaction should commit.
    *
    */
+  @Order(2)
   @Test
   @DisplayName("Check cross domain transaction with TMAfterTLogBeforeCommitExit property commits")
   public void testCrossDomainTransactionWithFailInjection() {
 
-    logger.info("Getting admin server external service node port");
-    int domain1AdminServiceNodePort = assertDoesNotThrow(
-        () -> getServiceNodePort(domain1Namespace, domain1AdminServerPodName + "-external", "default"),
-        "Getting admin server node port failed");
-
     String curlRequest = String.format("curl -v --show-error --noproxy '*' "
             + "http://%s:%s/cdttxservlet/cdttxservlet?namespaces=%s,%s",
-        K8S_NODEPORT_HOST, domain1AdminServiceNodePort, domain1Namespace, domain2Namespace);
+        K8S_NODEPORT_HOST, admin1ServiceNodePort, domain1Namespace, domain2Namespace);
```

Comment:
I like the original name - domain1AdminServiceNodePort better since it tells me immediately that it is the admin server belonging to domain1

Reply:
done